instructions/1-your-first-component.md

```diff
@@ -265,7 +265,7 @@ export class Counter extends Component {
     return (
       <div>
         <h2>Count: {this.state.count}</h2>
-        <button type="button" onClick={this.incrementCounter}>increment</button>
+        <button type="button" onClick={() => this.incrementCounter()}>increment</button>
```

Comment:
Yep, my bad. If you want to be really efficient, you'd rather use 

```javascript
this.incrementCounter = this.incrementCounter.bind(this);
```

in the constructor and keep `onClick={this.incrementCounter}` in the `<button />`